spark/v3.2/spark/src/test/java/org/apache/iceberg/spark/sql/TestSelect.java

```diff
@@ -49,8 +49,8 @@ public TestSelect(String catalogName, String implementation, Map<String, String>
 
   @Before
   public void createTables() {
-    sql("CREATE TABLE %s (id bigint, data string, float float) USING iceberg", tableName);
-    sql("INSERT INTO %s VALUES (1, 'a', 1.0), (2, 'b', 2.0), (3, 'c', float('NaN'))", tableName);
+    sql("CREATE TABLE %s (id bigint, data string, float float, binary binary) USING iceberg", tableName);
+    sql("INSERT INTO %s VALUES (1, 'a', 1.0, X''), (2, 'b', 2.0, X'1111'), (3, 'c', float('NaN'), X'11')", tableName);
```

Comment:
Please don't edit existing test cases. Instead, add new test cases/methods for what you are changing.

Reply:
ok, thanks for your comment